### PR TITLE
Add MedischeZending

### PR DESCRIPTION
**Pull request by le-jun**
Suriname’s health operator with more than 50 centers.

### CODE REVIEW

File: data/operators/amenity/clinic.json [hunk 1]
```diff
@@ -646,6 +646,19 @@
...
       }
     },
+    {
+      "displayName": "Medische Zending",
+      "locationSet": {"include": ["sr"]},
+      "tags": {
+        "amenity": "clinic",
+        "healthcare": "clinic",
+        "operator": "Medische Zending Primary Health Care Suriname",
+        "operator:short": "MZPHCS;MZ;Medische Zending",
```
Comment (LaoshuBaby) [inline data/operators/amenity/clinic.json:656]: Maybe the more common used "Medische Zending" can be operator, not short? 

This is only my personal suggestion, we still respect local mapper's opinion.

Comment (le-jun) [reply data/operators/amenity/clinic.json:656]: Doing so, would there be an appropriate key for the operator official full name?

Comment (LaoshuBaby) [reply data/operators/amenity/clinic.json:656]: for default name there are `alt_name/official_name` variants , add a `"^.*_"` before `"name"`.

but for `brand`, `operator` and other things, "short" and "official" are surfix to describe the namespace. 

So in OSM's naming habit, I guess "operator:official" maybe the best key. Although there haven't enough element using this key:

图片

**Review by le-jun — COMMENTED** [review]
Modified operator and operator:official keys

File: data/operators/amenity/clinic.json (hunk 1 above)
Comment (le-jun) [reply data/operators/amenity/clinic.json]: Well, that’s fine by me

**Pull request merged**